### DISCUSSION TITLE
Major new release

### DISCUSSION THREAD

**Discussion by agronick** [agronick/aa-torque#60] [Announcements]
There is a new release that is a prerelease. It will not trigger the update in the app. Test it out and report if you have any issues.

New chart feature to graph your gauges. 
New translations.

**Full Changelog**: https://github.com/agronick/aa-torque/compare/10.0.71...10.0.73

<hr /><em>This discussion was created from the release <a href='https://github.com/agronick/aa-torque/releases/tag/10.0.73'>13</a>.</em>

**Comment by mkwatchface** [comment]
Hello,

Thanks for keeping up with the work on this! The updates are much appreciated! One comment in this pre-release, it seems that if you have selected to disable the buttons and use the rotary input to change dashboards, the graph button is not available.

Is this intended?

Thanks

**Reply by agronick** [reply]
Press down on the rotary input to change views.

**Reply by mkwatchface** [reply]
Hello, pressing the rotary button didn't work for me...

**Reply by agronick** [reply]
Download and install this release and paste the logs. It will show what keycode is being sent: https://github.com/agronick/aa-torque/releases/tag/10.0.75

**Comment by Taryro55** [comment]
thank you so much for this, cheers

**Comment by agronick** [comment]
Guys if you can get the word out, post to whatever forums you're a part of or YouTube, Facebook, Insta showing off your setup it would be much appreciated.

**Reply by Vince56190** [reply]
Done here ( my channel) : https://youtu.be/ijNiJ47pz5A
Done here ( my group) :https://www.facebook.com/groups/LanciaThema
Done here ( my forum) : https://lancia-thema-france.forumactif.com/t452-jauges-type-srt-personalisable
And Here : https://www.300cforums.com/threads/torque-app-for-android-auto.283319/

**Comment by kacnje** [comment]
I posted about AA Torque on [my blog](https://www.kacnje.eu/blog) as you probably remember. And guess what - the post is visited quite often. 
Super. Just go on.
